### ISSUE TITLE
FB00000804 (Redo)

### ISSUE THREAD

**Issue by robes** [informatics-isi-edu/facebase-curation#41]
This was the GEO dataset submitted before as just a link to GEO. We now have the data and a (geo) metadata spreadsheet.

data-staging:/home/vasu/GSE67985*

**Comment by robes** [comment]
I just took another look to refresh my memory. What we have are a .tar file with a batch of gzipped cel files. Then we have a GEO text / spreadsheet for their GEO metadata. What we can do is load the gzipped cel files. But the spoke will have to curate the metadata according to the facebase standards.

**Comment by crisaless** [comment]
https://www.facebase.org/chaise/record/#1/isa:dataset/RID=VX6

still no data files included

**Comment by bugacov** [comment]
I believe we decided that on datasets like this we'll just leave the dataset with the link to GEO. 
@robes .... what do you think? 
@BridgetDS ... do you think it will make sense to try to do additional curation on a dataset like this and upload the data files?

**Comment by BridgetDS** [comment]
We've just been leaving them on geo only, and adding the link to geo as a related dataset. Not sure if this will be confusing for users.

**Comment by robes** [comment]
Yes, we can just reference the GEO dataset. A good description of the dataset on FaceBase would be important but the files and data might as well remain where they’ve already been submitted.

On 2/5/20, 6:05 PM, "bugacov" <notifications@github.com<mailto:notifications@github.com>> wrote:


I believe we decided that on datasets like this we'll just leave the dataset with the link to GEO.
@robes<https://github.com/robes> .... what do you think?
@BridgetDS<https://github.com/BridgetDS> ... do you think it will make sense to try to do additional curation on a dataset like this and upload the data files?

**Comment by BridgetDS** [comment]
Note that these "GEO-only" datasets get flagged for curation issues:
<img width="1302" alt="Screen Shot 2020-02-07 at 11 13 45 AM" src="https://user-images.githubusercontent.com/14893197/74058463-f2788f80-499a-11ea-9f8a-a5c3f72ff904.png">

**Comment by robes** [comment]
Good point. Let’s bring this up with Alejandro. He can adjust the curation QC rules so that we make these an exception to the normal expectations.

On 2/7/20, 11:15 AM, "BridgetDS" <notifications@github.com<mailto:notifications@github.com>> wrote:


Note that these "GEO-only" datasets get flagged for curation issues:
[Image removed by sender. Screen Shot 2020-02-07 at 11 13 45 AM]<https://user-images.githubusercontent.com/14893197/74058463-f2788f80-499a-11ea-9f8a-a5c3f72ff904.png>
